From 47d2101eb4aa13a0624cd1783cffaf2b9c7d6f82 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 23 Mar 2020 09:58:29 -0700
Subject: [PATCH 1290/1852] drm: xlnx: bridge: make xlnx_bridge_set_timing()
 static inline
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Without inline, it gives below compile warning.

drivers/gpu/drm/xlnx/xlnx_bridge.h:149:12: warning: ‘xlnx_bridge_set_timing’ defined but not used [-Wunused-function]
 static int xlnx_bridge_set_timing(struct xlnx_bridge *bridge,
		             ^~~~~~~~~~~~~~~~~~~~~~

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_bridge.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_bridge.h b/drivers/gpu/drm/xlnx/xlnx_bridge.h
index 64330169bd22..fb1b4b748346 100644
--- a/drivers/gpu/drm/xlnx/xlnx_bridge.h
+++ b/drivers/gpu/drm/xlnx/xlnx_bridge.h
@@ -146,8 +146,8 @@ static inline int xlnx_bridge_get_output_fmts(struct xlnx_bridge *bridge,
 	return 0;
 }
 
-static int xlnx_bridge_set_timing(struct xlnx_bridge *bridge,
-				  struct videomode *vm)
+static inline int xlnx_bridge_set_timing(struct xlnx_bridge *bridge,
+					 struct videomode *vm)
 {
 	if (bridge)
 		return -ENODEV;
-- 
2.31.1

